read-fonts/src/tables/varc.rs

```diff
@@ -215,26 +192,115 @@ impl<'a> VarcComponent<'a> {
             axis_values,
             axis_values_var_index,
             transform_var_index,
-            dx,
-            dy,
-            rotation,
-            sx,
-            sy,
-            skewx,
-            skewy,
-            center_x,
-            center_y,
+            transform,
         })
     }
 }
 
+/// <https://github.com/fonttools/fonttools/blob/5e6b12d12fa08abafbeb7570f47707fbedf69a45/Lib/fontTools/misc/transform.py#L410>
+pub struct DecomposedTransform {
+    translate_x: f64,
+    translate_y: f64,
+    rotation: f64, // degrees, counter-clockwise
+    scale_x: f64,
+    scale_y: f64,
+    skew_x: f64,
+    skew_y: f64,
+    center_x: f64,
+    center_y: f64,
+}
+
+impl Default for DecomposedTransform {
+    fn default() -> Self {
+        Self {
+            translate_x: 0.0,
+            translate_y: 0.0,
+            rotation: 0.0,
+            scale_x: 1.0,
+            scale_y: 1.0,
+            skew_x: 0.0,
+            skew_y: 0.0,
+            center_x: 0.0,
+            center_y: 0.0,
+        }
+    }
+}
+
+impl DecomposedTransform {
+    /// Convert decomposed form to standard 2x3 matrix form
```

Comment:
iirc there is some variation in the world in terms of how the fields of an affine matrix are ordered, and I think it's worth being explicit about the field order in the documentation (see [kurbo, for example](https://docs.rs/kurbo/latest/kurbo/struct.Affine.html#method.new))

Reply:
Oh good point. I tend to think of kurbo-style (a b are x-basis, c d are y-basis, e f are translation) as standard but it is not at all true.